packages/mobile/src/screens/edit-profile-screen/EditProfileScreen.tsx

```diff
@@ -172,8 +172,8 @@ export const EditProfileScreen = () => {
     tiktok_handle,
     website,
     donation,
-    cover_photo: { url: coverPhotoSource[0].uri },
-    profile_picture: { url: imageSource[0].uri }
+    cover_photo: { url: coverPhotoSource[1].uri },
+    profile_picture: { url: imageSource[2].uri }
```

Comment:
This is kinda gross, basically selecting the appropriate size from an array of sizes. I can add a nicer utility around this but for now we can stick with this quick fix